### ISSUE TITLE
YOLO11 got stuck during pruning while generating pruner

### ISSUE THREAD

**Issue by TenMinutes209** [VainF/Torch-Pruning#454]
When I tried to prune an YOLO11 model with Backbone replaced with ShuffleNetV2, I got problems below:

**The code stuck in the sentence** 
pruner = tp.pruner.GroupNormPruner(
            model.model,
            example_inputs,
            importance=tp.importance.GroupNormImportance(),  # L2 norm pruning,
            iterative_steps=1,
            pruning_ratio=pruning_ratio,
            ignored_layers=ignored_layers,
            unwrapped_parameters=unwrapped_parameters
        )

How can I solve this problem? Thank you

**Comment by TenMinutes209** [comment]
I tried that when I **keep the param.requires_grad = False**, the code could run but the model wasn't pruned. When I turned **param.requires_grad = True**, the code stuck.

**Comment by loserking111** [comment]
Have you successfully pruned YOLO11
